website/content/docs/nia/compatibility.mdx

```diff
@@ -13,10 +13,12 @@ The following tables list the Consul-Terraform-Sync (CTS) version compatibility
 
 Below are CTS versions with supported Consul versions. The latest CTS binary supports the three most recent Consul minor versions, along with their latest patch versions.
 
-| CTS Version         | Consul OSS & Enterprise Version |
-| :------------------ | :------------------------------ |
-| CTS Enterprise 0.3+ | 1.8+, HCP-Consul                |
-| CTS OSS 0.1+        | 1.8+, HCP-Consul                |
+| CTS Version         | Consul OSS & Enterprise Version | HCP Consul Version |
+| :------------------ | :------------------------------ | :----------------- |
+| CTS Enterprise 0.6+ | 1.8+                            | 1.9+               |
+| CTS Enterprise 0.3+ | 1.8+                            | N/A                |
+| CTS OSS 0.6+        | 1.8+                            | 1.9+               |
+| CTS OSS 0.1+        | 1.8+                            | N/A                |
```

Comment:
Similarly here, wondering if it makes sense to update this to something like `CTS OSS 0.1-0.5`